dhall-docs/tasty/data/golden/InvalidConsecutiveComments.dhall.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE HTML>
+<html
+  ><head
+    ><title
+    >/InvalidConsecutiveComments.dhall</title
+    ><link href="index.css" type="text/css" rel="stylesheet"
+    /><link href="https://fonts.googleapis.com/css2?family=Fira+Code:wght@400;500;600;700&amp;family=Lato:ital,wght@0,400;0,700;1,400&amp;display=swap"
+            type="text/css" rel="stylesheet"
+    /><script src="index.js" type="text/javascript"
+    /><meta charset="UTF-8"/></head
+  ><body
+    ><div class="nav-bar"
+      ><img src="dhall-icon.svg" class="dhall-icon"
+      /><p class="package-title"
+      >test-package</p
+      ><div class="nav-bar-content-divider"
+      /><a id="switch-light-dark-mode" class="nav-option"
+      >Switch Light/Dark Mode</a></div
+    ><div class="main-container"
+      ><h2 class="doc-title"
+        ><span class="crumb-divider"
+        >/</span
+        ><a href="index.html"
+        >test-package</a
+        ><span class="crumb-divider"
+        >/</span
+        ><span href="index.html" class="title-crumb"
+        >InvalidConsecutiveComments.dhall</span></h2
+      ><a data-path="/InvalidConsecutiveComments.dhall"
+          class="copy-to-clipboard"
+        ><i
+          ><small
+          >Copy path to clipboard</small></i></a
+      ><br
+      /><div class="doc-contents"
+      /><h3
+      >Source</h3
+      ><div class="source-code"
+        ><pre
+          ><span class="dhall-syntax"
+          >{</span
+          ><span class="dhall-syntax"
+          >=</span
+          ><span class="dhall-syntax"
+          >}</span></pre></div></div></body></html>
```

Comment:
nit: would it be possible to ensure that the generated files end with a carriage return?

Reply:
These files are not directly generated by the tool, they are the result of generating the HTML Text documentation from the package and passing that result to `HaXml` parser and pretty printer. I guess that a solution would be to append a newline to the result of the prettyprinter.